layouts/partials/case-studies.html

```diff
@@ -1,37 +1,30 @@
-{{ $data := index site.Data site.Language.Lang }}
-
-{{ if $data.casestudies.casestudies.enable }}
-{{ with $data.casestudies.casestudies }}
 <section class="sm:py-9">
-    <h5 class="font-bold uppercase text-[#757575]">{{ .tagline }}</h5>
-    <h2 class="py-2 text-2xl uppercase md:text-4xl font-extrabold">{{ .title }}</h2>
-    <p class="py-4 md:text-lg">{{ .subtitle }}</p>
+    {{ with .Params.expertise }}
+    <h5 class="font-bold uppercase text-[#757575]">{{ .subheading }}</h5>
+    <h2 class="py-2 text-2xl uppercase md:text-4xl font-extrabold">{{ .heading }}</h2>
+    <p class="py-4 md:text-lg">{{ .description }}</p>
+    {{ end }}
     <section class="grid md:grid-cols-2 lg:grid-cols-3 gap-8 py-8 text-center">
-        {{ range .studies }}
-        <a href="{{ .link }}">
-            <div class="flex flex-col h-full bg-[#E3E3E1] pb-4 rounded-md">
-                <div class="mb-4 md:mb-4">
-                    <h4 class="text-lg md:text-xl font-bold py-6">{{ .summary}}</h4>
-                    <div class="md:h-36 inline-flex items-center mb-6 md:mb-3">
-                        <img src="{{ .image }}" alt="" class="mx-auto" />
-                    </div>
+        
+      {{ $paginator := .Paginate .Data.Pages }}
+      {{ range $paginator.Pages.ByWeight }}
+        <a href="/case-studies/{{ .Params.slug }}">
+            <div class="flex flex-col h-full border bg-[#ECF6FF] py-4 rounded-md">
```

Comment:
The box color was changed following a discussion with Edwin. The industry and service type were added to provide more context to users and so that the boxes wouldn't look so empty.